src/5e-SRD-Monsters.json

```diff
@@ -20935,7 +21475,84 @@
     "actions": [
       {
         "name": "Multiattack",
-        "desc": "The lizardfolk makes two melee attacks, each one with a different weapon."
+        "desc": "The lizardfolk makes two melee attacks, each one with a different weapon.",
+        "options": {
+          "choose": 1,
```

Comment:
Geeze, I wish we had a good schema shape for "Choose two distinct". This is probably fine though?